cluster/watchdog.go

```diff
@@ -61,6 +64,8 @@ func (w *Watchdog) rescheduleContainers(e *Engine) {
 	defer w.Unlock()
 
 	log.Debugf("Node %s failed - rescheduling containers", e.ID)
+
+reschduleLoop:
```

Comment:
I am not sure if this label `reschduleLoop ` is necessary, as out of  `continue reschduleLoop` , there is only one `for` loop. @dongluochen